core/src/messaging/framing.rs

```diff
@@ -295,7 +295,7 @@ fn system_path_put_into_buf(path: &SystemPath, buf: &mut dyn BufMut) -> Result<(
         IpAddr::V6(ref ip) => buf.put_slice(&ip.octets()),
         // TODO support named Domain
     }
-    buf.put_u16_be(path.port());
+    buf.put_u16(path.port());
```

Comment:
Side note: it's so weird to me that they would default to big-endian in the `bytes` crate. Like, I know the crate is targeted mostly towards networking code, but the first line of their documentation doesn't even mention that. It just says that the crate "provides abstractions for working with bytes". Not using the native byte order by default seems like a weird abstraction to me.

Reply:
I'm sure there is a long discussion somewhere about this...but in the end it makes sense...you wanna optimise your API for the common case and most people serialise in BE for compatibility.